cmd/plugins_test.go

```diff
@@ -313,3 +348,18 @@ func TestLambdaFileWalker(t *testing.T) {
 		}
 	})
 }
+
+func signalHandler(tempRootDir string, cleanup func()) {
+	signalChan := make(chan os.Signal, 1)
+	signal.Notify(signalChan, syscall.SIGINT, syscall.SIGTERM)
+	go func() {
+		s := <-signalChan
+		switch s {
+
+		case syscall.SIGINT, syscall.SIGTERM, syscall.SIGQUIT:
+			fmt.Println("Received an ", s.String(), "cleaning ", tempRootDir, "and exiting")
```

Comment:
[nit] What would you say to using `fmt.Printf` instead? 😉

Reply:
This could be in a OPA style guide.